config.toml

```diff
@@ -29,20 +29,20 @@ pluralizelisttitles = false
     weight = -200
 
 [[menu.main]]
-	  name   = "Download"
-		weight = 600
+    name   = "Download"
+    weight = 600
 
 [[menu.main]]
-	  parent = "Download"
+    parent = "Download"
     name   = "KiCad Software"
     url    = "/download/"
     weight = 400
 
 [[menu.main]]
     parent = "Download"
-		name   = "KiCad Libraries"
-		url    = "/libraries/"
-		weight = 300
+    name   = "KiCad Libraries"
+    url    = "/libraries/"
+    weight = 300
```

Comment:
I think having software up top (smaller weight) might be preferable as that's probably the primary thing the user is looking for when clicking on the menu.